src/main/include/CobConstants.h

```diff
@@ -0,0 +1,6 @@
+#pragma once
+
+enum class CobKey {
```

Comment:
Cob setup looks great. How would enums serialize to a file?

Reply:
They are easily convertible to unique ints with static_cast<> [stackoverflow](https://stackoverflow.com/questions/8357240/how-to-automatically-convert-strongly-typed-enum-into-int)
Also we may not need to write our own serialization system if SetPersistent works as expected